package.json

```diff
@@ -174,9 +174,11 @@
     "ws": "^7.3.0",
     "xml2js": "^0.4.23"
   },
+  "optionalDependencies": {
+    "@serverless/components": "^2.30.11"
```

Comment:
I think we cannot do that, as this dependency is used upfront by the CLI.

Also internally it's assumed as required. Concept of `optionalDependencies` is more for dependencies which cannot be installed on some platforms and package has workarounds to work without them, here it's not the case

Reply:
Understood